### PR TITLE
API : update doc related to issue #1956

### PR DESCRIPTION
**Pull request by foxmask**

### CODE REVIEW

File: readme/api.md [hunk 1]
```diff
@@ -119,6 +119,9 @@ Gets note with ID :id
 
 Gets all the tags attached to this note.
 
+## GET /notes/:id/resources
+
+Gets all the resources attached to this note.
```
Comment (tessus) [inline readme/api.md]: To be consistent with the rewst f the file, there should be an empty line between the text and the next header.

**Comment by tessus** [comment]
Thanks for the update.

**Comment by laurent22** [comment]
By the way, the API doc is auto-generated by the api-doc command, but that's fine I've moved your changes there: https://github.com/laurent22/joplin/commit/3a32e790014f63bade55fb971dafeb1a15bc0e86

**Comment by tessus** [comment]
Thx, I totally forgot about that.

**Pull request merged**